packages/core/src/amazonqGumby/activation.ts

```diff
@@ -13,62 +13,59 @@ import { ProposedTransformationExplorer } from '../codewhisperer/service/transfo
 import { CodeTransformTelemetryState } from './telemetry/codeTransformTelemetryState'
 import { telemetry } from '../shared/telemetry/telemetry'
 import { CancelActionPositions } from './telemetry/codeTransformTelemetry'
-import { AuthUtil } from '../codewhisperer/util/authUtil'
 
 export async function activate(context: ExtContext) {
     void vscode.commands.executeCommand('setContext', 'gumby.wasQCodeTransformationUsed', false)
-    // If the user is codewhisperer eligible, activate the plugin
-    if (AuthUtil.instance.isValidCodeTransformationAuthUser()) {
```

Comment:
The only change is that this line has been removed; the rest of the changes are formatting / whitespace related

Reply:
Does this mean that the Transformation Hub will always show up, even for users who haven't interacted with QCT? I remember adding this line a while ago because we received some feedback that the Transformation Hub shouldn't show up unless a user has interacted with QCT.

Reply:
I tested this scenario, let me know if it covers the edge case you're concerned about:

1. Open VSCode, sign into Amazon Q
2. I go to the lower windows (where terminal, etc) are located. `Transformation Hub` is not visible
3. I click `...` but `Transformation Hub` is not an option there
4. It is only when I successfully submit a project to be transformed that `Transformation Hub` becomes a tab in this collection of lower windows
5. If I close `Transformation Hub` after it is open, I can only open it from the Chat (selecting `Show Transformation Hub`)